tests/projects/service/deployment/project.yml

```diff
@@ -1,7 +1,7 @@
 name: 'nodeservice'
 description: 'This is a test container. For testing the MPL pipelines, not to be deployed anywhere.'
 stages:
-  build: Echo Build
+  build: ''
```

Comment:
Can we change the schema so that `build` and `test` are optional fields instead?

Reply:
They are optional, but I wanted to keep this field (even though the step doesn't exist anymore) for the plan detection logic

Reply:
Ah, so now as long as the field is there the service is seen as having a build stage?

Reply:
Always been this way for the plan detection logic. Only when you do an actual run it will error saying it can't find a step called ''